app/views/admin/users/_form.html.erb

```diff
@@ -244,6 +244,8 @@
   </div>
   <% end %>
 
+  <% if !@user.is_administrator? %>
```

Comment:
Why is this change in the PR and what is the reason for it?  If not an admin show the save button??  Seems odd.

Reply:
I think I was originally thinking the oauth admin permission would be on the user and this was the start of that, but I found out that would not work and forgot I added the line. It is removed.